### DISCUSSION TITLE
DDOSed my IP on a hobby plan ☹️

### DISCUSSION THREAD

**Discussion by mrlowe** [vercel/community#7014] [Account Issue] [answered]
### Question

I have a project on a hobby plan that I am planning to move to a professional plan, but before I did that I ran a little local testing with k6 to start figuring out how big a server I'd need. Apparently I tripped DDOS protection at the firewall level, because now from this IP I'm getting PR_END_OF_FILE_ERROR. Other IPs can access the app instance just fine, and I'm also able to access the dashboard at vercel.com just fine.

Any advice on how to get my IP unbanned for this application (aside from pulling a new IP from my ISP)?

Also, is there anything in the Vercel dashboard I missed that lets me warn a server "I'm getting ready to load test you, don't ban me"?

**Comment by stuffthatspins** [comment]
similar issue, I was crawling my site to create an XML sitemap.  I don't know how to remove my IP from the DDos list.

**Reply by mrlowe** [reply]
Since we don't seem to be getting any Vercel response, I'll share my experience: after a few days -- maybe as long as a week -- I was able to make requests to the server from my office machine again. I don't think its IP changed, but it's not static so of course that's possible.

**Comment by amyegan** [comment, marked as answer]
Hi there! [Load testing is only permitted on **Enterprise** plans](https://vercel.com/guides/what-s-vercel-s-policy-regarding-load-testing-deployments) and must be coordinated with the our Customer Success team. 

[Vercel WAF](https://vercel.com/docs/security/vercel-waf) can restrict access to certain IP addresses. If you need to modify the [Vercel WAF IP Blocking](https://vercel.com/docs/security/vercel-waf/ip-blocking#waf-ip-blocking) settings for your project, follow these steps:

1. Navigate to the Firewall tab of your project.
2. Select Configure on the top right of the Firewall overview page.
3. Scroll down to the IP Blocking section.
4. Find the IP address that you want to unblock and remove it from the list.

If you're not seeing your IP address in the IP Blocking section, it's possible that your IP address was blocked due to a security measure. You might need to contact your network administrator or ISP for further assistance.

If none of that works for you, please [submit a case](https://vercel.com/help) for further help with your IP or project settings.

**Reply by mrlowe** [reply]
Ah, that information helps a lot, thank you. For me the answer is "you buy an Enterprise plan and contact Customer Success." Which is unfortunate, because the point of load testing at this point is to help me understand what plan we need.

> If none of that works for you, please [submit a case](https://vercel.com/help) for further help with your IP or project settings.

I think we can't submit a case on a hobby plan, but that may be helpful for others.

**Reply by amyegan** [reply]
Our sales team is always happy to talk about project requirements to help determine the best plan for you. They can coordinate with other teams to make sure you have the info you need.

[vercel.com/contact/sales](https://vercel.com/contact/sales)
